olavra/unity-renderer
# Feat/procedural skybox horizon plane

## What does this PR change?



Horizon Plane is added to the skybox shader and tool

## How to test the changes?



1. Go to:  https://play.decentraland.org/?renderer=https://decentraland-olavra.s3.eu-west-1.amazonaws.com/branch/feat/Procedural_Skybox-HorizonPlane
2.  Changes to the genrix config is required to see the effect in the build.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
